### ISSUE TITLE
Installer v0.3.1 error: "[blank] wants to make changes."

### ISSUE THREAD

**Issue by Indenex** [microsoft/vscode-java-installer#2]
Program asking for permission to install says "[blank] wants to make changes."
![2020-07-09_17-00-48](https://user-images.githubusercontent.com/15364122/87095141-d3aa4d80-c205-11ea-8dbf-2f8c607e0ee3.png)

version: 0.3.1
env:
```
os: darwin
arch: x64
PATH: /usr/bin:/bin:/usr/sbin:/sbin
JAVA_HOME: undefined
JDK_HOME: undefined
```
log:
```
[4:57:34 PM] AdoptOpenJDK-11-x64
[4:58:24 PM] Download JDK ... [Completed]
[4:58:24 PM] Install JDK
[4:58:40 PM] Error occurred
[4:58:40 PM] Error: User did not grant permission.

```

**Comment by Eskibear** [comment]
Thanks for reporting this, it's AdoptOpenJDK asking for permissions. We'll fix that.

**Comment by Eskibear** [comment]
It's a bug of Catalina 10.15.5, and would be fixed in 10.15.6
See https://github.com/jorangreef/sudo-prompt/issues/127
